src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json

```diff
@@ -6,49 +6,56 @@
     "email" : "alice@example.com",
     "dob" : "1/1/2000",
     "address" : "123, Jurong West Ave 6, #08-111",
-    "tagged" : [ "friends" ]
+    "tagged" : [ "friends" ],
+    "gender" : "Male"
```

Comment:
I think for sake of easier testing can put some of the typicalPersons genders as female as well